engine/src/main/java/org/terasology/rendering/dag/nodes/CopyImageToScreenNode.java

```diff
@@ -60,12 +65,15 @@ public void initialise() {
     public void process() {
         PerformanceMonitor.startActivity("rendering/copyImageToScreen");
         sceneFinalFbo.bindTexture(); // TODO: Convert to a StateChange
```

Comment:
Okay, so I remembered why I could not convert this to a SetInputTextureFromFBO call earlier.

SetInputTextureFromFbo expects a slot, an fbo an attachment type, a materialUrn and a materialParameter. fbo.bindTexture() operates on an fbo, and uses the ColorBufferTextureId. The missing component between bindTexture() and SITFF is the slot (which I assume will be 0 for color?), the materialUrn (no idea) and the materialParameter.

Hence, I gave up mid-conversion.